root@2b5e05e0812a:/workspaces/Safe-Reinforcement-Learning-for-Black-Box-Systems-Using-Reachability-Analysis# cd catkin_ws/
root@2b5e05e0812a:/workspaces/Safe-Reinforcement-Learning-for-Black-Box-Systems-Using-Reachability-Analysis/catkin_ws# source devel/setup.bash
root@2b5e05e0812a:/workspaces/Safe-Reinforcement-Learning-for-Black-Box-Systems-Using-Reachability-Analysis/catkin_ws# rosrun brsl Turtlebot_environment.py
2024-05-16 22:29:15.942485: I tensorflow/core/util/port.cc:110] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-05-16 22:29:15.949834: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-05-16 22:29:16.020932: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-05-16 22:29:16.021565: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-05-16 22:29:17.064869: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
/usr/local/lib/python3.8/dist-packages/tf_agents/specs/array_spec.py:335: RuntimeWarning: invalid value encountered in cast
  self._minimum[self._minimum == -np.inf] = low
/usr/local/lib/python3.8/dist-packages/tf_agents/specs/array_spec.py:336: RuntimeWarning: invalid value encountered in cast
  self._minimum[self._minimum == np.inf] = high
/usr/local/lib/python3.8/dist-packages/tf_agents/specs/array_spec.py:338: RuntimeWarning: invalid value encountered in cast
  self._maximum[self._maximum == -np.inf] = low
/usr/local/lib/python3.8/dist-packages/tf_agents/specs/array_spec.py:339: RuntimeWarning: invalid value encountered in cast
  self._maximum[self._maximum == np.inf] = high
shutdown request: [/environment] Reason: new node registered with same name
root@2b5e05e0812a:/workspaces/Safe-Reinforcement-Learning-for-Black-Box-Systems-Using-Reachability-Analysis/catkin_ws# 